srcpkgs/timeshift/template

```diff
@@ -1,20 +1,22 @@
 # Template file for 'timeshift'
 pkgname=timeshift
-version=22.06.1
+version=22.11.2
 revision=1
 build_style=gnu-makefile
 hostmakedepends="gettext pkg-config vala which"
 makedepends="libgee08-devel json-glib-devel gtk+3-devel vte3-devel libgirepository-devel"
 depends="rsync psmisc"
 short_desc="System restore tool"
 maintainer="Orphaned <orphan@voidlinux.org>"
-license="GPL-3.0-or-later"
-homepage="https://teejeetech.com/timeshift/"
-distfiles="https://github.com/teejee2008/timeshift/archive/v${version}.tar.gz"
-checksum=aef3980c19dfde9084b47b4e04e5106489ee7ddf44e6228415d2521405a091ec
+license="GPL-3.0-only"
+homepage="https://github.com/linuxmint/timeshift"
+changelog="https://raw.githubusercontent.com/linuxmint/timeshift/master/debian/changelog"
+distfiles="
+ https://github.com/linuxmint/timeshift/archive/${version}/timeshift-${version}.tar.gz"
+checksum=36fd19865d1d30dc5f6eca468ceaa3dad01cd95d9090d050d67f5c2cd98fe37d
 
-# Remove hard-dep on crontab, leave up to user to decide
-post_extract() {
-	sed -i '/dependencies/s:"crontab",::' src/Core/Main.vala
-	sed -i 's:--thread:& Core/AppExcludeEntry.vala:' src/makefile
```

Comment:
why remove these?

Reply:
ugh... right. I removed cronie and forgot to put this back. FIxed.